partials/archive-artist.dust

```diff
@@ -0,0 +1,65 @@
+{>"shared/base" /}
+
+{<content}
+    {#ArchiveArtist}
+        <main class="main-content section" id="main-content">
+            <div class="container">
+                {>"shared/archive-view-title" /}
+                {>"shared/archive-filters" /}
+
+                <div class="columns is-multiline artist-list">
+                    {#results}
+                        <div class="column is-6-tablet is-3-desktop mb-6-desktop is-relative has-text-centered">
+                            {?image}
+                                <div class="artist-list__image-container image is-3by1 has-width-100 pr-6 pl-6 mb-4">
+                                    <div
+                                        class="artist-list__image is-absolute has-top-0 has-bottom-0 has-right-0 has-left-0" {@inlinebg id=image size="large" /}></div>
+                                </div>
+                            {/image}
+
+                            <div class="artist-list__text-grid">
+                                {@isset key1=fields.is_concertmaster key2=fields.is_principal method="OR" }
```

Comment:
Tämän logiikan olisi kaiketi voinut tehdä myös PHP:n puolella, jolloin ei dustin päässä tarttisi. Mutta jos toimii näin hyvin niin ok.